client/src/components/behavior/chart/Rechart.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+import { ResponsiveContainer, Line, LineChart, Tooltip, Legend, YAxis, XAxis, CartesianGrid } from 'recharts';
+
+export default class Chart extends Component {
```

Comment:
We Should use functional components - we also already have a component named `<Chart />` so this needs to be exported as something else.

Also, perhaps a clearer component name - I'm not sure what Rechart implies here

Reply:
I changed it to BehavioralChart